pic-build

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
+#
+# Copyright 2017 Axel Huebl
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+this_dir=`dirname $0`
+build_dir=".build"
+
+help()
+{
+    echo "Build new binaries for a PIConGPU input set"
+    echo ""
+    echo "Creates or updates the binaries in an input set. This step needs to"
+    echo "be performed every time a .param file is changed."
+    echo ""
+    echo "This tools creates a temporary build directory, configures and"
+    echo "compiles current input set in it and installs the resulting"
+    echo "binaries."
+    echo "This is just a short-hand tool for switching to a temporary build"
+    echo "directory and running 'pic-configure ..' and 'make install'"
+    echo "manually."
+    echo ""
+    echo "You must run this command inside an input directory."
+    echo ""
+    echo "usage: pic-build [OPTIONS]"
+    echo ""
+    # lend the rest of the options from pic-configure
+    echo "$($this_dir/pic-configure --help | tail -n +12)"
```

Comment:
if we do so, which we don't do often, the output of `pic-build --help` will just start a bit in-the-middle.

nevertheless, it's better than re-typing the exact same options. also one would see it quickly if forgotten, since it's output ins included in the basic usage in the manual.